errbot/botplugin.py

```diff
@@ -266,6 +266,19 @@ def callback_message(self, message: Message) -> None:
         """
         pass
 
+    def callback_mention(self, message: Message, mentioned_people: Sequence[Identifier]) -> None:
+        """
+            Triggered if there are mentioned people in message.
```

Comment:
Good point! I agree with you @zoni.
Maybe we can provide dummy default implementation for other backends - for example if mentioning is not supported by some backend, we can "fake it" mentioning user from `message.frm` or `message.to`. How this sounds to you?